src/main/java/shop/service/impl/OrderServiceImpl.java

```diff
@@ -1,6 +1,52 @@
 package shop.service.impl;
 
+import shop.dao.OrderDAO;
+import shop.dao.impl.OrderDAOImpl;
+import shop.entity.Order;
 import shop.service.OrderService;
 
-public class OrderServiceImpl {
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class OrderServiceImpl implements OrderService {
+    private Map<String, Order> orderMap = new HashMap<>();
+    private OrderDAO orderDAO = new OrderDAOImpl();
+
+    @Override
+    public void addProductToOrder(String productId) {
+
+        orderMap.put(productId, (Order) orderDAO);
```

Comment:
it should be an update of existing order. Create a method update() in OrderDAO

if user wants to add some product to order:
we need to get his order from a map, modify object Order (by adding a new element to list productList()
and then, put it again to the map by key - to replace existing